src/components/pokedex.html

```diff
@@ -250,7 +250,14 @@ <h4 data-bind="text: `Showing ${PokedexHelper.filteredList().length} Pokémon`">
                                                         (App.game.party.getPokemon($data.id)?.pokerus >= GameConstants.Pokerus.Contagious ?
                                                             `<br/>EVs: <strong>${App.game.party.getPokemon($data.id).evs().toLocaleString('en-US')}</strong>
                                                             <br/>EV bonus: <strong>×${App.game.party.getPokemon($data.id).calculateEVAttackBonus().toLocaleString('en-US')}</strong>`
-                                                            : ''),
+                                                            : '') +
+                                                        (PokemonHelper.hasMegaEvolution($data.name) ?
+                                                            App.game.party.getPokemon($data.id).attack < pokemonMap[$data.name].attack * GameConstants.MEGA_REQUIRED_ATTACK_MULTIPLIER ?
+                                                                `<br/>Needs at least ${(pokemonMap[$data.name].attack * GameConstants.MEGA_REQUIRED_ATTACK_MULTIPLIER).toLocaleString('en-US')} attack to Mega Evolve.`
+                                                                : PokemonHelper.getMegaStones($data.name).some(m => player.hasMegaStone(m.megaStone)) ?
+                                                                    '<br/>Use a Key Stone to Mega Evolve.'
+                                                                    : `<br/>Obtain the Mega Stone to Mega Evolve ${$data.name}.`
```

Comment:
Can this not just use the hint from the mega evolution instead?

Reply:
The hint is on the MegaEvolveRequirement so it's not the cleanest but I guess it could be done like this:
`pokemonMap['Pidgeot'].evolutions.flatMap(e => e.restrictions).find(r => r instanceof MegaEvolveRequirement).hint()`.
The only caveat is for Charizard and Mewtwo, who have two mega evos, it would only ever mention the first stone.

Reply:
Maybe for each mega evolution, there could be a header with the mega stone name and the evo requirement hint underneath?

Reply:
Alright, tweaked it to do just that. It's not pretty but it works. Also added the tooltip back to the pokemon list icon. 

![image](https://user-images.githubusercontent.com/672420/236707662-69a4de15-2b74-486d-b6a3-0da2390204bd.png)
![image](https://user-images.githubusercontent.com/672420/236707669-da2f1b4d-47ba-40c9-88a3-52ac4d6e219b.png)